### ISSUE TITLE
javaType does not generate class when used with annotationStyle=gson

### ISSUE THREAD

**Issue by kkuo** [joelittlejohn/jsonschema2pojo#668]
With something like this:
```

{
    "javaType": "com.company.Ipv4Port",
    "type":"object",
    "properties": {
        "ip": {
            "type":"string"
        },
        "port": {
            "type":"string"
        }
}
```

generates a Ipv4Port class with annotationStyle set to default, but does not generate a class when using annotationStyle=gson

**Comment by joelittlejohn** [comment]
This works for me with your example. Is it because you haven't cleaned the old generated types before trying a second time? If there is already a class on the classpath with the name specified in javaType then no new class will be generated.

**Comment by kkuo** [comment]
looks like this is fixed in 0.4.9. I was using 0.4.0.
